curvefs/src/client/inode_wrapper.h

```diff
@@ -363,6 +377,19 @@ class InodeWrapper : public std::enable_shared_from_this<InodeWrapper> {
 
     CURVEFS_ERROR RefreshVolumeExtent();
 
+    bool NeedRefreshData() {
+        uint64_t size = GetS3ChunkSize();
```

Comment:
Can we cache the number of s3info of current inode? so, we don't have to calculate the size every time.

For example, when we fetch chunkinfo map from metaserver, we can calculate an initial value, and in `AppendS3ChunkInfo` adds 1 to the value. So, `GetS3ChunkSize` can directly returns the cached value.

This logic can also apply to `GetS3ChunkInfoAddSize`

Reply:
fixed, recorded in inodeWrapper and update when append and refresh